test/functional/tools/exit_code_from_env.xml

```diff
@@ -2,7 +2,7 @@
     <!-- tool errors out with identified OOM error if less than 10MB are allocated. -->
     <command detect_errors="exit_code" oom_exit_code="42"><![CDATA[
 echo 'Hello' > '$out_file1' &&
-: \${GX_TARGET_EXIT_CODE:-0} &&
+: \${GX_TARGET_EXIT_CODE:=0} &&
```

Comment:
Why is this change needed? 

Out of curiosity: What is the difference between `-` and `=` here?

Reply:
If I remember well, with a `-` the default value was not assigned to the variable => not used in the next line (exit)

Some more detail here: https://unix.stackexchange.com/questions/122845/using-a-b-for-variable-assignment-in-scripts